tour/exercise-1.md

```diff
@@ -198,8 +198,7 @@ u:http://oldnews.com/a23 uri hash	a6c4d1f
 
 Now that you have data loading, create an observer that watches the reference
 status column.  This observer should increment word counts when new content is
-referenced and decrement word counts when content is dereferenced.  The
-observer should also delete the content when its dereferenced.
+referenced and decrement word counts when content is dereferenced.  The observer should also delete the content if zero URLs reference this one. If content's URL reference counts goes to zero and it was processed before, then its word counts should be decremented before deleting.
```

Comment:
I have been trying to think of way to make this prose easier to understand and I could not.  I think replacing the prose with a table would make this easier to understand.  What do you think of the following?

```markdown
Now that you have data loading, create an observer that watches the reference
status column (*doc:refs*).  This observer should take the following actions 
based on the values of the columns *doc:refs* and *doc:processed*.  

| doc:refs     | doc:processed       | Action
| ------------ | ------------------- | ----------
| referenced   | false               | Increment word counts and set *doc:processed* to *true*
| referenced   | true                | None
| unreferenced | false               | Delete content
| unreferenced | true                | Delete content and decrement word counts
```

@llvieira this is not the changes I was thinking of when I opened that issue (I'll update the issue to clarify).  However that issue I opened was awful, it took me a while to figure out what I was thinking.  Sorry!  Anyway the changes you are making here are great because this is really confusing and needs to be improved.

Reply:
@keith-turner I think this table is awesome. I will update the `exercise-1.md`.